RecoBTag/SecondaryVertex/python/secondaryVertex_cff.py

```diff
@@ -46,8 +46,10 @@
 from RecoBTag.SecondaryVertex.pfGhostTrackVertexTagInfos_cfi import *
 from RecoBTag.SecondaryVertex.pfGhostTrackBJetTags_cfi import *
 from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderAK8TagInfos_cfi import *
+from RecoBTag.SecondaryVertex.pfBoostedDoubleSVAK8TagInfos_cfi import *
 from RecoBTag.SecondaryVertex.pfBoostedDoubleSecondaryVertexAK8BJetTags_cfi import *
 from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderCA15TagInfos_cfi import *
+from RecoBTag.SecondaryVertex.pfBoostedDoubleSVCA15TagInfos_cfi import *
```

Comment:
Can these `import *` statements be avoided? It is better to import only what is needed.